letsencrypt-auto-source/letsencrypt-auto.template

```diff
@@ -534,6 +546,24 @@ if [ "$1" = "--le-auto-phase2" ]; then
   # Phase 2: Create venv, install LE, and run.
 
   shift 1  # the --le-auto-phase2 arg
+
+  if [ "$DEPRECATED_OS" = 1 ]; then
+    # Phase 2 damage control mode for deprecated OSes.
+    # In this situation, we bypass any bootstrap or certbot venv setup.
+    error "Your system is not supported by certbot-auto anymore."
+
+    if [ -f "$VENV_BIN/letsencrypt" ]; then
```

Comment:
I just noticed that this code breaks the `--install-only` flag on deprecated systems. To fix this, how about we also check `"$INSTALL_ONLY" != 1` here and go into the "Certbot cannot be installed" case.

I think it'd be good to add a simple test for this case where an older version of Certbot is already installed too.

Reply:
Also, nit:

I think we should add a comment before the `INSTALL_ONLY` case below saying something like:

> If you're modifying any of the code after this point in this if block, you may need to update the "$DEPRECATED_OS" = 1 case at the beginning of phase 2 as well.

Reply:
Done.